test/testinput/increment.yml

```diff
@@ -1,5 +1,38 @@
 geometry:
```

Comment:
The format of the yaml file here doesn't look right. It shouldn't have `state test`, It should have `increment test`. see https://github.com/JCSDA-internal/soca/blob/develop/test/testinput/increment.yml

Reply:
ah, ok i see `increment test` at the bottom. Remove the `state test` section, and place a `tolerance` in `increment test` in order to get all the tests passing

Reply:
The tolerance for Increment is defined in the source file "bundle/oops/src/test/interface/Increment.h" line 66 as 
"const double tol_default = 1e-8;"
Should we change the source file?

Reply:
It will pass the test if we change it to 1e-7. I just git push the new increment.yml.

Reply:
No, that's the default tolerance. The tolerance is overridden by putting `tolerance: ` in your `increment test` ,the same way that was done for `state test`. Increasing the tolerance used should make that one failing test pass, figure out what value is required for that to happen

Reply:
Yes it works, just updated the increment.yml. Now it should be ok.